src/Commands/EasterEggs.py

```diff
@@ -47,39 +92,40 @@ def Teams() -> str:
             "A team: doesn't practice but somehow is good"
             "\nB team: everyone hates how their teamates play but don't talk it out to resolve issues"
             "\nC team: who?"
-            "\nD team: best team"
+            "\nD team: H Team, Best Team™"
             "\nE team: surprisingly solid"
             "\nF team: how many fkn teams do we have"
             "\nGG team: originally g team")
 
 
 def NormQ() -> List[str or Embed]:
-    messages: List[str or Embed] = []
     playerList = Queue.getQueueList()
     queueSize = Queue.getQueueLength()
 
-    messages.append("Duis says I am not supposed to queue, but I don't listen to players worse than me...")
-    messages.append("!q")
-
     if (Queue.queueAlreadyPopped() or queueSize == 6):
-        messages.append(ErrorEmbed(
+        blueTeam, orangeTeam = Queue.getTeamList()
+        blueCap, orangeCap = Queue.captainsPop()
+        embed = ErrorEmbed(
             title="Current Lobby Not Set",
             desc="Whoa there Norm! You can't queue until the current queue has finished popping."
-        ))
+        )
+        CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
```

Comment:
Shouldn't this be `updated_embed = CaptainsRandomHelpEmbed(...)` like we talked about?

Reply:
Resolved.